hedera-node/src/test/java/com/hedera/test/utils/SeededPropertySource.java

```diff
@@ -474,6 +474,12 @@ public ExpirableTxnRecord nextRecord() {
 		if (seeded.getContractCreateResult() != null && nextBoolean()) {
 			seeded.getContractCreateResult().setSenderId(nextEntityId());
 		}
+		// added in 0.28
+		if (nextBoolean()) {
+			seeded.setPseudoRandomNumber(nextUnsignedInt());
+		} else {
```

Comment:
We can make this `else if (nextBoolean()) {` so that some of the records don't have any PRNG output.

Reply:
Makes sense. Fixed !